### PR TITLE
Download page with 4 columns

### PR DESCRIPTION
**Pull request by alfrix**

### CODE REVIEW

**Comment by alfrix** [comment]
@twinaphex did i change too much, want me to split it up?

File: platforms.php [hunk 1]
```diff
@@ -38,217 +30,233 @@
...
+            -->
+            <div class="col-md-3 col-sm-6">
+               <div class="platform">
+                 <img src="images/xbox.png">
+                 <h4>Xbox</h4>
+                 <a href="https://xbins.org/libretro/stable/<?php echo $version; ?>/xbox/xbox/RetroArch.zip">Download [Xbins]</a>
+               </div>
+            </div>
+        </div>
+        <div class="row platforms">
```
Comment (RobLoach) [inline platforms.php]: Shouldn't need new rows for each one.... Just stick all the cells in progression and Bootstrap will fix the rows for you.

File: platforms.php [hunk 2]
```diff
@@ -38,217 +30,233 @@
...
                 </div>
-                <div class="col-md-3 col-sm-6">
-                  <div class="platform">
+            </div>
+        </div>
+        <div class="row platforms">
+            <div class="col-md-3 col-sm-6">
+               <div class="platform">
+                   <img src="images/linux.png">
+                   <h4>Linux</h4>
```
Comment (RobLoach) [inline platforms.php]: Looks like the order of the platforms is changing. If I were making this change, I'd take that change in a different PR. Then it'll be easier to review :wink:

File: platforms.php [hunk 3]
```diff
@@ -38,217 +30,233 @@
...
+                   <s><a href="https://buildbot.libretro.com/stable/<?php echo $version; ?>/apple/osx/x86/RetroArch_CG.dmg" title="Use this if you are using a 32bit Mac with at least MacOS X 10.6 installed or later. This build requires Nvidia Cg installed, and will use it. If in doubt, use the other one.">Download (Intel 32bit) (with Cg support - requires Cg installed)</a></s>
+                   <a href="https://buildbot.libretro.com/stable/<?php echo $version; ?>/apple/osx/ppc/RetroArch.zip" title="Use this if you are using a PowerMac/Powerbook with MacOS X 10.5 installed.">Download (PowerPC)</a>
+                   <a href="https://docs.libretro.com/compilation/osx/">Compilation instructions (Intel)</a>
+                   <a href="https://docs.libretro.com/compilation/osxppc/">Compilation instructions (PowerPC)</a>
+                   <p>Note: OSX version for x86 32bit will return at a later point in time. </p>
+                </div>
+            </div>
+        </div>
+        <div class="row platforms">
```
Comment (RobLoach) [inline platforms.php]: As above, don't need a new `row platforms`, just have all the cells in sequence.

**Comment by twinaphex** [comment]
OK, let's try this.

**Pull request merged**